### ISSUE TITLE
Chapter 7: Where is the dataset

### ISSUE THREAD

**Issue by andysingal** [Apress/computer-vision-projects-with-pytorch#4]
I was working on Chapter but was not able to find the dataset. Do you think you can provide with the information.
        Here is the training data:
        tr_ds_path = "/content/drive/MyDrive/car_img/tr" #Training Images
        Here is the validation data:
        vd_ds_path = "/content/drive/MyDrive/car_img/vds" #Validation Images
        Here is the test data:
        ts_ds_path = "/content/drive/MyDrive/car_img/ts" #Test Images

**Comment by nitinranjansharma** [comment]
This is a mixed dataset created from https://www.kaggle.com/competitions/carvana-image-masking-challenge/data this few other class added to let the model figure out whether thats an anomaly or not. You can use fruits360 data along with this to test it out
